### PR TITLE
Johnson ogwuru

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js [hunk 1]
```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+export default function CardContainer({ content, method }) {
+    return (
+        <div className="cardSection" onClick={()=>{window.location.href="https://www.reactjs.org"}} >
```
Comment (mattshardman) [inline Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js]: I'm not sure why you wouldn't just use an ```href``` here but Melvine did the same so maybe I've missed something?!

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+            <div className="cardWrapper">
+                <CardBanner image = { content.image } />
+                <CardContent title={content.title} link={content.link} content = { content.content }/>
+            </div>
```
Comment (mattshardman) [inline Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js]: You could use the spread operator to pass these in i.e. 

```jsx
<CardContent {...content} />
```

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js (hunk 1 above)
Comment (ogwurujohnson) [reply Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js]: Have gotten use to doing it like that ,  would try out just  href and push

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js (hunk 2 above)
Comment (ogwurujohnson) [reply Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js]: Would do immediately

**Comment by mattshardman** [comment]
Nice work Johnson!

- [ ]  Your social card closely resembles the design files
- [ ]  Your git work flow is excellent
- [ ]  Your have separated the card into separate components logically
- [ ]  You have gone above the MVP by adding in a footer component with social icons

**Possible things to work on**
- Couple of comments above but just general advice really!

Overall really good job, everything is implemented correctly. You clearly have the hand of functional components in React

**Pull request merged**